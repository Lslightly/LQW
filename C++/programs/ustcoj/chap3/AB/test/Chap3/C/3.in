uhfuhdqopcfxchaqhjbc
uhfuhdqopcthchaqhabc
uzfuhdqopcthmhaqhcbc
ujfuhdqopcyhmhaqhcbg
ujfuhjqopcyhmhachcbg
ujfuhjqopcyhmhachdaz
uifuhjqopcuhmhachdan
uifuhjqopcuhjhakxdan
uifuhjqppcupjhakxdab
uifuhjqpwcupjhfkxdfb
uifuhjqpwcupjifkiafb
uifshjqpwcupjifkiafb
uibsmjqpwkupjifkiafb
uibsmjqxwkupsifkiafb
uibsmjvxykupsifhiafb
uibsmjmlykupsifhiwfb
uibsmemrbkupsifhiwfb
uibsmemrbvppscfhiwfb
uiysmemrbvppscfhiwfe
uirsmemgbvppscahiwfe
urmsmemwbvppscahiwfe
armsmemwbvppscahqrfe
rrmsmemwbvpphcahqcfe
rrmsmexwbvpphcahqcve
rrmvmexwxvpphcahqsve
rrmvmexwivphhcahqsve
rrmvmexkijphhcahqsve
rrmvmexkijuhhcloqsve
rlmvmexkiwunhcloqsve
rlmnmexkiwunhcloaeve
rlmnmsxkiwunhceoheve
rlmnmspoiwunhceohevu
rvmnmspomwunhceoievu
rvmnmsygmwuthceoievu
avbnmsygmwutcceoievu
avmnmsigmwuacceoievu
avmnmsignwuacceoaevu
avmnmsignwugjceoaefu
avmnmgignwugjcloayfu
vvmnmgignwugjcloyyfu
yvmnggignwugjcloyqfu
yvmnggignwugjcboyefu
yvmngibgnwubjcboyefu
yfnngibgnwubjcboyefp
yfghgibgywubjcboyefp
jfghgyboywubjcboyefp
jfghgeboywubjcboyehp
jfghgeboywxbjcfoyehu
jfghgeboywhbjcfoyezv
jfhhjeboywhbjcfxyezv
