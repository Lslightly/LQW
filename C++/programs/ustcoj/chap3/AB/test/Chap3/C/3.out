uhfuhdqopcfxchaqhjbc uhfuhdqopcf
uhfuhdqopcthchaqhabc uhfuhdqopct
uzfuhdqopcthmhaqhcbc uz
ujfuhdqopcyhmhaqhcbg ujfuhd
ujfuhjqopcyhmhachcbg ujfuhjqopcyhmhachc
ujfuhjqopcyhmhachdaz ujfuhjqopcyhmhachd
uifuhjqopcuhmhachdan uifuhjqopcuhm
uifuhjqopcuhjhakxdan uifuhjqopcuhj
uifuhjqppcupjhakxdab uifuhjqpp
uifuhjqpwcupjhfkxdfb uifuhjqpwcupjh
uifuhjqpwcupjifkiafb uifuhjqpwcupji
uifshjqpwcupjifkiafb uifs
uibsmjqpwkupjifkiafb uibsmjqp
uibsmjqxwkupsifkiafb uibsmjqx
uibsmjvxykupsifhiafb uibsmjv
uibsmjmlykupsifhiwfb uibsmjm
uibsmemrbkupsifhiwfb uibsmemrbk
uibsmemrbvppscfhiwfb uibsmemrbv
uiysmemrbvppscfhiwfe uiy
uirsmemgbvppscahiwfe uir
urmsmemwbvppscahiwfe ur
armsmemwbvppscahqrfe ar
rrmsmemwbvpphcahqcfe rrmsmem
rrmsmexwbvpphcahqcve rrmsmex
rrmvmexwxvpphcahqsve rrmvmexwx
rrmvmexwivphhcahqsve rrmvmexwi
rrmvmexkijphhcahqsve rrmvmexkijp
rrmvmexkijuhhcloqsve rrmvmexkiju
rlmvmexkiwunhcloqsve rlmv
rlmnmexkiwunhcloaeve rlmnme
rlmnmsxkiwunhceoheve rlmnmsx
rlmnmspoiwunhceohevu rlmnmsp
rvmnmspomwunhceoievu rvmnmsp
rvmnmsygmwuthceoievu rvmnmsy
avbnmsygmwutcceoievu avb
avmnmsigmwuacceoievu avmnmsigm
avmnmsignwuacceoaevu avmnmsignwua
avmnmsignwugjceoaefu avmnmsignwug
avmnmgignwugjcloayfu avmnmg
vvmnmgignwugjcloyyfu v
yvmnggignwugjcloyqfu yvmnggignwugjcl
yvmnggignwugjcboyefu yvmnggignwugjcb
yvmngibgnwubjcboyefu yvmngi
yfnngibgnwubjcboyefp yfn
yfghgibgywubjcboyefp yfg
jfghgyboywubjcboyefp jfghgy
jfghgeboywubjcboyehp jfghgeboywu
jfghgeboywxbjcfoyehu jfghgeboywx
jfghgeboywhbjcfoyezv jfghgeboywh
jfhhjeboywhbjcfxyezv jfh
